lib/datasource/remote/polkadot_api/chains_repository.dart

```diff
@@ -98,4 +103,63 @@ class ChainsRepository {
     final list = await getChainsLocal();
     return list.where((e) => e.header.toLowerCase() != "polkadot" && e.header.toLowerCase() != "kusama").toList();
   }
+
```

Comment:
this logic creates the list of networks - it was previously in the use case, moved it here